module-with-no-tests/.gitignore

```diff
@@ -0,0 +1,38 @@
+target/
+!.mvn/wrapper/maven-wrapper.jar
+!**/src/main/**/target/
+!**/src/test/**/target/
+
+### IntelliJ IDEA ###
+.idea/modules.xml
```

Comment:
if anything this belongs into a root .gitignore.

Reply:
it is intellij auto-generated .gitignore.To be honest I did not even open it. I'll improve it. Thanks

Reply:
Done